src/distributed-sql/dist-sql.md

```diff
@@ -0,0 +1,88 @@
+# Distributed SQL 
+
+Now we already know how the TiDB relational structure are encoded into the Key-Value form with versions.  In this section, we will focus on the following questions:
+
+* What happens when [TiDB] receives a SQL query?
+* How does [TiDB] excute SQL queries in a distributed way?
+  
+## What happens when [TiDB] receives a SQL query?
+
+Firstly, let's have a look at the following example:
+
+```
+   select count(*) from t where a + b  > 5;
+```
+
+![Figure 1](images/select_from_tidb.png)
+
+
+As the above figure discribes, when [TiDB] receives a SQL query from the client, it will process according the following steps:
+
+1. [TiDB] receives a new SQL from the client.
+2. [TiDB] prepares the process plans for this request, meanwhile [TiDB] get a tso from [PD] as the transaction's `start_ts`.
+3. [TiDB] tries to get information schema(meta data of the table) from cache, get from TiKV instead if it not exist.
```

Comment:
There is something wrong in this sentence, but I don't know how to express it :(